extensions/markdown/syntaxes/markdown.tmLanguage

```diff
@@ -2404,6 +2431,38 @@
 						</dict>
 					</array>
 				</dict>
+				<key>fenced_code_block_rust</key>
```

Comment:
Please update these to use the fenced code block logic that the other languages now use. It's rather horrific but it fixes a few important edge cases